eureka/S3_data_reduction/background.py

```diff
@@ -198,108 +197,4 @@ def fitbg(dataim, mask, x1, x2, deg=1, threshold=5, isrotate=False, isplots=Fals
         bg   = (bg.T)
         mask = (mask.T)
 
-    return bg, mask #,variance
-
-# STEP 3: Fit sky background with out-of-spectra data
-def fitbg2(dataim, mask, bgmask, deg=1, threshold=5, isrotate=False, isplots=False):
```

Comment:
`fitbg2` will likely be needed for NIRISS data reduction, so best to leave it in.

Reply:
Okay, is there some more detailed information I can put into the docstrings to specify why one might choose fitbg2 over fitbg?

Reply:
fitbg2 uses bgmask, a mask for the background region! It enables fitting more complex BG regions than simply above or below a given distance from the trace.  This will help mask the 2nd and 3rd orders of NIRISS.